sympy/codegen/ast.py

```diff
@@ -151,7 +193,7 @@ def aug_assign(lhs, op, rhs):
         subclass these types are also supported.
 
     op : str
-        Operator (+, -, /, *, %).
+        Operator (+, -, /, \*, %).
```

Comment:
In Python 3.6, backslash escapes that don't actually escape anything are a syntax warning, and are tests should fail when they encounter them (it was broken, but I fixed it in https://github.com/sympy/sympy/pull/13083). In a future version of Python it will be a syntax error. You can make the docstring a raw string to avoid the issue.